sat-plugin/src/main/resources/rulesets/checkstyle/rules.xml

```diff
@@ -234,6 +234,8 @@
       <property name="severity" value="info" />
     </module>
     <module name="MethodName">
+        <property name="format"
+            value="^\_?[a-z][a-zA-Z0-9]*$"/>
```

Comment:
Should we just keep those info's as know defects to always 'promote' underscore feels a little bit to broad

Reply:
Should we make them warnings then? Or errors? The other option, which will not 'promote' underscore is to suppress them (only 2 files).

Reply:
No, info is fine, but just not allow the underscore and indeed suppressing for those two files sounds more sane then changing the regex.